src/libraries/Microsoft.Bcl.Cryptography/src/Resources/Strings.resx

```diff
@@ -171,6 +171,12 @@
   <data name="Cryptography_KeyWrongSizeForAlgorithm" xml:space="preserve">
     <value>The specified key is not the correct size for the indicated algorithm.</value>
   </data>
+  <data name="Cryptography_MLDsaNoSecretKey" xml:space="preserve">
+    <value>The current instance does not contain a secret key.</value>
+  </data>
```

Comment:
Secret Key vs Decapsulation Key means that to share this with ML-KEM we'd have to parameterize it.  But MLDsaNoSeed and MLKemNoSeed seem pretty sharable.  Do we really need the same string twice under two names?

Reply:
I can put this in a future PR to avoid merge conflicts since `MLKemNoSeed` is still in PR pending merge. I like the logical separation of ML-KEM and ML-DSA but it is unfortunate that it would result in more resources in the dll.

Reply:
Mine's not even approved yet, so if you want to rename yours to something like `Cryptography_PqcNoSeed` I can merge in `main` when you merge and then use the de-duped name.